developer-docs-site/docs/guides/prover-guides/spec-lang.md

```diff
@@ -1367,24 +1201,21 @@ spec hash {
 spec fun abstract_hash(v: vector<u8>): u64; // uninterpreted function
 ```
 
-The soundness of the abstraction is the responsibility of the specifier, and not verified by the
-prover.
+The soundness of the abstraction is the responsibility of the specifier and not verified by the
+Move Prover.
 
-> NOTE: the abstract/concrete properties should only be used with opaque specifications, but the prover will
-> currently not generate an error if not.
+> NOTE: The abstract/concrete properties should only be used with opaque specifications, but the Move Prover will currently not generate an error if not.
```

Comment:
Please clarify this note. When we say, "but the Move Prover will currently not generate an error if not" if not what?